lib/ecto/repo/registry.ex

```diff
@@ -18,9 +18,8 @@ defmodule Ecto.Repo.Registry do
   end
 
   def lookup(repo) when is_atom(repo) do
-    GenServer.whereis(repo)
-    |> Kernel.||(raise "could not lookup Ecto repo #{inspect repo} because it was not started or it does not exist")
-    |> lookup()
+    :persistent_term.get(repo) ||
```

Comment:
Double checking: I think get/1 either returns :undefined or raises so wondering if we should use `get(repo, nil)` instead. If we are setting it prior, nevermind!

Reply:
Yes, that's what we wanted, thank you!!!

Reply:
```suggestion
    :persistent_term.get(repo, nil) ||
```